inspirehep/modules/theme/bundles.py

```diff
@@ -68,14 +68,9 @@
     }
 )
 
-detailedjs = NpmBundle(
-    "js/detailed_record_init.js",
-    filters="requirejs",
-    output="gen/detailed_record.%(version)s.js",
-)
-
 css = NpmBundle(
     "scss/inspirehep.scss",
+    "node_modules/datatables.net-dt/css/jquery.dataTables.css",
```

Comment:
How to make this bootstrap-capable.
`"node_modules/datatables.net-bs/css/dataTables.bootstrap.css",`